frontend/javascripts/navbar.js

```diff
@@ -300,6 +300,18 @@ function LoggedInAvatar({ activeUser, handleLogout, ...other }) {
         <Menu.Item disabled key="organization">
           {orgDisplayName}
         </Menu.Item>
+        <Menu.Item key="switch">
```

Comment:
This does not really seem to work for me. When I click it (in Firefox) the menu closes, so I can’t type in it to see what it does. Also, this should probably only appear in the isMultiMember case, right?
From the code, I don’t really understand what it is supposed to do.

Reply:
This was not supposed to be part of the PR. Just a failed experiment. Would be nice to have something like this, tho :D